external/pdal_wrench/translate.cpp

```diff
@@ -0,0 +1,172 @@
+/*****************************************************************************
+ *   Copyright (c) 2023, Lutra Consulting Ltd. and Hobu, Inc.                *
+ *                                                                           *
+ *   All rights reserved.                                                    *
+ *                                                                           *
+ *   This program is free software; you can redistribute it and/or modify    *
+ *   it under the terms of the GNU General Public License as published by    *
+ *   the Free Software Foundation; either version 3 of the License, or       *
+ *   (at your option) any later version.                                     *
+ *                                                                           *
+ ****************************************************************************/
+
+#include <iostream>
+#include <filesystem>
+#include <thread>
+
+#include <pdal/PipelineManager.hpp>
+#include <pdal/Stage.hpp>
+#include <pdal/util/ProgramArgs.hpp>
+#include <pdal/pdal_types.hpp>
+#include <pdal/Polygon.hpp>
+
+#include <gdal/gdal_utils.h>
+
+#include "utils.hpp"
+#include "alg.hpp"
+#include "vpc.hpp"
+
+using namespace pdal;
+
+namespace fs = std::filesystem;
+
+
+void Translate::addArgs()
+{
+    argOutput = &programArgs.add("output,o", "Output point cloud file", outputFile);
+    argOutputFormat = &programArgs.add("output-format", "Output format (las/laz/copc)", outputFormat);
+    programArgs.add("assign-crs", "Assigns CRS to data (no reprojection)", assignCrs);
+    programArgs.add("transform-crs", "Transforms (reprojects) data to anoter CRS", transformCrs);
```

Comment:
```suggestion
    programArgs.add("transform-crs", "Transforms (reprojects) data to another CRS", transformCrs);
```

Reply:
Fixed in https://github.com/PDAL/wrench/commit/7e5b28746dac6380e5afa8d87288f9d9dceb4fac